compiler/qsc_eval/src/tests.rs

```diff
@@ -2148,6 +2148,121 @@ fn update_range_to_end() {
     check_expr("", "...2..3 w/ End <- 10", &expect!["...2..10"]);
 }
 
+#[test]
```

Comment:
These tests are great! I wasn't expecting support for open-ended ranges, but it makes sense. Can you add a test for using the fully open range: `...`?

Reply:
oh, good idea! I forgot about the fully open ended case.